governance-exercises/VoteWizard_poll_exercise_02/Derecognition of Clive AD Due to Compromised Operational Security - July 28, 2023.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Derecognition of Clive AD Due to Compromised Operational Security - JULY 28, 2023
+summary: Signal your support or opposition to derecognition of Clive AD due to compromised operational security. A community member, Diane, posted materials claiming that they could reveal the delegate's identity.
+discussion_link: https://forum.makerdao.com/t/...
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-28T16:00:00
+end_date: 2023-07-31T16:00:00
+---
+# Poll: Derecognition of Clive AD due to compromised operational security - JULY 28, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) has been posted as per [MIP106c2.6.6.2.1A](https://mips.makerdao.com/mips/details/MIP106#6-6-current-incubating-ecosystem-actors) and will be active for three days beginning on Monday, July 28 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the potential derecognition of Clive AD due to compromised operational security.
+Materials posted by a community member, Diane, claim that they could reveal the delegate's identity. As per the [Atlas](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy), if these materials constitute significant suspicion, then the aligned delegate has to be derecognized.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/...) and the materials provided by Diane in [this post](https://forum.makerdao.com/t/...) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* The Aligned Delegate will be derecognized.
+* Half of the Aligned Delegate Buffer will be confiscated and used as the whistleblower bounty.
```

Comment:
Arguably this is a separate concern, and should be dealt with by the facilitator without being constricted by the result of the vote. 

Reason for this is that Diane might have been a complete dick about the whole thing, to the point where voters want to disincentivise that behaviour, but also see the need to get rid of Clive.